cabbage/tasks.py

```diff
@@ -21,20 +24,39 @@ def __init__(
         self.func: Callable = func
         self.name = name or self.func.__name__
 
-    def defer(self, lock: str = None, **kwargs: types.JSONValue) -> None:
+    def defer(
+        self,
+        lock: str = None,
+        scheduled_time: Optional[datetime] = None,
+        **kwargs: types.JSONValue,
+    ) -> None:
         lock = lock or f"{uuid.uuid4()}"
+        scheduled_time = scheduled_time or pendulum.now()
+
+        if scheduled_time.tzinfo is None:
+            # Make sure we have an explicit timezone on the schedule time
+            # We consider a naive datetime to be in the local timezone
```

Comment:
What about forbidding a naive datetime ?

```console
$ python -m this | sed '14q;d'
In the face of ambiguity, refuse the temptation to guess.
```

Reply:
That would be ideal, yes.